ako-operator/config/samples/ako_v1alpha1_akoconfig.yaml

```diff
@@ -6,10 +6,11 @@ metadata:
   name: ako-sample
   namespace: avi-system
 spec:
-  imageRepository: projects.registry.vmware.com/ako/ako:1.8.2
+  imageRepository: projects.registry.vmware.com/ako/ako:1.9.1
```

Comment:
Do we have to give 1.9.3 instead of 1.9.1 here?

Reply:
Have updated to 1.9.3